packages/calcite-components/src/components/shell-panel/shell-panel.tsx

```diff
@@ -75,8 +75,13 @@ export class ShellPanel implements ConditionalSlotComponent, LocalizedComponent,
   }
 
   /**
-   * Specifies the display mode - `"dock"` (full height, displays adjacent to center content), `"float"` (not full height, content is separated detached from `calcite-action-bar`, displays on top of center content),
-   * or `"overlay"` (full height, displays on top of center content).
+   * Specifies the display mode of the component, where:
+   *
+   * `"dock"` full height, displays adjacent to center content,
+   *
+   * `"float"` not full height, content is separated detached from `calcite-action-bar`, displays on top of center content, and
+   *
+   * `"overlay"` full height, displays on top of center content.
```

Comment:
nitpick: The re-org makes some of the wording sound off, WDYT about resorting a bit to something like:

> `"dock"` displays at full height adjacent to center content,
> `"float"` does not display at full height with content separately detached from `calcite-action-bar` on top of center content, and
> etc...